src/components/DatasetPreviewCard.tsx

```diff
@@ -34,7 +35,7 @@ interface IState {}
 
 class DatasetPreviewCard extends React.Component<IProps, IState> {
   render(): ReactElement {
-    const { classes, name, description, updatedAt = 'error', tags = ['is_pii'] } = this.props
+    const { classes, name, description, updatedAt, tags = ['is_pii'] } = this.props
```

Comment:
Should default tags by `[]`?

Reply:
yeah I'm down with that, this is legacy from playing around :-|